### ISSUE TITLE
Support [Object] Array for New-UDTable Data Custom Column Renders

### ISSUE THREAD

**Issue by JustinGrote** [ironmansoftware/powershell-universal-legacy#68]
#### Description

This used to work in 2.x when the data was provided as a json rather than object info

If you set the data as an array of objects instead of an array of hashtables the custom column rendering will not work. Often the data being used for tables is not in hashtable format and it shouldn't be required to first convert it to hashtables first.

#### Reproduction
![Capture](https://user-images.githubusercontent.com/15258962/97237091-b93fb400-17a3-11eb-94d4-6049ad81f6bd.gif)

#### Workaround
If PS6+ Dashboard, you can do this to quickly convert the object into a hashtable:
```powershell
$ObjectDataArray | ConvertTo-Json -Depth 99 | ConvertFrom-Json -AsHashTable
```

#### Internal Issue

We'll fill this out when we create an internal issue in our repository.

**Comment by adamdriscoll** [comment]
This is fixed in 1.4.6.

**Comment by JustinGrote** [comment]
@adamdriscoll confirmed fixed. Thanks!
